tests/integ/modin/series/test_rank.py

```diff
@@ -29,7 +29,8 @@
 
 
 @sql_count_checker(query_count=1)
-@pytest.mark.parametrize("data, index", TEST_RANK_DATA)
+# Skipping last test case since it uses MultiIndex.
```

Comment:
has anything changed about the MultiIndex case here?

Reply:
Nope, this should still work. Thanks for catching that! I removed the slicing